catalog/registry/catalog-registry-cswinputtransformer/src/test/java/ddf/catalog/registry/transformer/RegistryTransformerTest.java

```diff
@@ -87,7 +112,7 @@ public void testOrgInfo() throws Exception {
 
         assertThat(meta.getOrgName(), is("Codice"));
         assertThat(meta.getOrgAddress(), is("1234 Some Street, Phoenix, AZ 85037, USA"));
-        assertThat(meta.getOrgPhoneNumber(), is("(555) 555-5555 extension 1234"));
```

Comment:
I originally removed the extension from the test xml because there weren't any test cases for phone without extension at the time. Phone without extension is being tested in other places now so I added it back.